Protos/V1/tasks_common.proto

```diff
@@ -76,6 +77,22 @@ message TaskSummary {
   string error = 8; /** The error message. Only set if task have failed. */
 }
 
+// TODO: Implements new filters using these fields.
+/**
+ * Represents every available field in a task summary.
+ */
+enum TaskSummaryField {
+  TASK_SUMMARY_FIELD_TASK_ID = 0; /** The task ID. */
+  TASK_SUMMARY_FIELD_SESSION_ID = 1; /** The session ID. */
+  TASK_SUMMARY_FIELD_STATUS = 2; /** The task status. */
+  TASK_SUMMARY_FIELD_CREATED_AT = 3; /** The task creation date. */
+  TASK_SUMMARY_FIELD_STARTED_AT = 4; /** The task start date. */
+  TASK_SUMMARY_FIELD_ENDED_AT = 5; /** The task end date. */
+  TASK_SUMMARY_FIELD_CREATION_TO_END_DURATION = 6; /** The task duration. Between the creation date and the end date. */
+  TASK_SUMMARY_FIELD_PROCESSING_TO_END_DURATION = 7; /** The task calculated duration. Between the start date and the end date. */
+  TASK_SUMMARY_FIELD_ERROR = 8; /** The error message. Only set if task have failed. */
+}
```

Comment:
I think we would also need a `TaskField` that would most likely be a `oneof` with `TaskSummaryField`, `TaskOptionFiled`, `TaskOptionGenericField`.

Reply:
On va attendre de supprimer le concept de task summary et je continue ensuite de bosser sur ça

Reply:
en attente de #255

Reply:
On ne supprime pas le concepte et je retourne bosser.